### ISSUE TITLE
Allow TVRenamer to access files stored on NAS

### ISSUE THREAD

**Issue by Riduidel** [tvrenamer/tvrenamer#325]
I would like to use TVRenamer from my desktop computer, with files stored on my NAS.
As this NAS support various protocols (sftp, SAMBA, and so on), it should be possible to access it from my computer. So is it possible to add such a feature ?

**Comment by Riduidel** [comment]
Having expressed my user need, let me tell you a few things.
To my mind, this feature could be quite easy to implement using [Commons-VFS](https://commons.apache.org/proper/commons-vfs/), which supports access to numerous file system types. I don't know if you use it already, and can provide some help to implement it.

**Comment by Litcube** [comment]
Yeah, I can't rename files that are sitting on my NAS.  No error message, it just doesn't rename them.

**Comment by Triforcecwp** [comment]
Ive come across the same issue. 
If i run it normally it can find the file but cannot rename it.
Running it as admin in windows 10 1803, it cannot even see my networked drives.

**Comment by littlediobolic** [comment]
I currently renaming all my shows that are sitting on my FreeNAS box from my Linux desktop over SFTP, seems to be working like a charm, at least for me.
